src/Sylius/Component/Association/README.md

```diff
@@ -0,0 +1,47 @@
+Association Component
+=====================
+
+The Sylius Association component efficiently manage product association for crosssell or upsell.
```

Comment:
Not only product, not only crosssell and upsell. It's generic component for any kind of associations between any objects.